ingestion-beam/README.md

```diff
@@ -296,6 +296,15 @@ gsutil cat $BUCKET/output/*
 
 ### On Dataflow with templates
 
+Dataflow templates make a distinction between
+[runtime parameters that implement the `ValueProvider` interface](https://cloud.google.com/dataflow/docs/guides/templates/creating-templates#runtime-parameters-and-the-valueprovider-interface)
+and compile-time parameters which do not.
+In the output of `--help=SinkOptions`, those options prefixed with `(runtime)`
+must be specified when
+[executing the template](https://cloud.google.com/dataflow/docs/guides/templates/executing-templates#using-gcloud)
+while all other parameters have to be specified as command line flags
+when compiling the template.
```

Comment:
This explanation doesn't match with @whd's understanding, so I need to run another test.